stdlib/2and3/zipfile.pyi

```diff
@@ -23,6 +24,25 @@ error = BadZipfile
 
 class LargeZipFile(Exception): ...
 
+class ZipExtFile(io.BufferedIOBase):
+    MAX_N: int = ...
+    MIN_READ_SIZE: int = ...
+
+    if sys.version_info < (3, 6):
+        PATTERN: Pattern = ...
+
+    if sys.version_info >= (3, 7):
+        MAX_SEEK_READ: int = ...
+
+    newlines: Optional[List[bytes]]
+    mode: str
+    name: str
+
+    def __init__(self, fileobj: IO[bytes], mode: str, zipinfo: ZipInfo, decrypter: Optional[Callable[[Sequence[int]], bytes]] = ..., close_fileobj: bool = ...) -> None: ...
+    def __repr__(self) -> str: ...
+    def peek(self, n: int = ...) -> bytes: ...
+    def read1(self, n: int) -> bytes: ...  # type: ignore
```

Comment:
The argument is actually `Optional[int]` since the implementation checks for None.